blockchain/contracts/DungeonDefenders/DefenderFactory.sol

```diff
@@ -4,9 +4,13 @@ pragma solidity ^0.8.0;
 import "../ContractUtils.sol";
 
 import "@openzeppelin/contracts/utils/Counters.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
 
-contract DefenderFactory is ContractUtils, Ownable {
+/// @title Defender Factory for DungeonDefenders
+/// @author rkhadder & 0xharb
+/// @notice Used by DefenderUtils.sol
+contract DefenderFactory is ContractUtils, AccessControl {
```

Comment:
Do we need to expose a method to assign roles? For example, the staking contract needs to have the Operator roll for DDS.

Reply:
Same question applies to DLOOT

Reply:
These methods are baked in into AccessControl
https://goerli.etherscan.io/address/0xfbcfbf4eea41de9a37ce976b035458886e8208f2#writeContract
![image](https://user-images.githubusercontent.com/30326647/177795437-9e951f4b-40e6-4a88-967a-8520590c83a8.png)

Documentation:
https://docs.openzeppelin.com/contracts/4.x/api/access#AccessControl-grantRole-bytes32-address-